### DISCUSSION TITLE
event conditional on watchlist of repository package dependencies

### DISCUSSION THREAD

**Discussion by yonicd** [community/community#26926] [Actions]
<p>Can I set a github action to be triggered based on a watchlist of repositories releases or changes to a subset of their branches (defaulting to only master)?</p>
<p>For example if my (R package) repository is dependent on 3 other repositories I would want to have my workflow be triggered when those repositories have an event on the master or some defined subset of branches.</p>
<p>One way to achieve this would be to have a list of repositories in the workflow event field.</p>
<p>this would be an example of how the event could be defined:</p>
<pre><code>event:
  observe:
    watch:
      - a
      - b: [master,dev]
      - c
    release: [a,b,c]
</code></pre>
<p>This would also get around a problem that “repository dispatch” has which you must have the right permissions to trigger that event.</p>
<p>With observe you would be passively triggering events instead of using an inefficient chron scheduling that can be used as a very blunt solution.</p>
<p>thanks!</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/yonicd">@yonicd</a> ,</p>
<p>It’s not supported to trigger a workflow based on a watchlist of repositories releases or changes.</p>
<p>Most of the events works in self repository scope, only ‘repository_dispatch’ is ‘external’ event which can trigger another repository.</p>
<p>Thanks.</p>

**Comment by yonicd** [comment]
<p>How is this marked as a solution? What did this answer solve?</p>
<p>Is there another forum for feature requests for github actions?</p>
<p>Thanks</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/yonicd">@yonicd</a> ,</p>
<p>Look it’s marked by another guy…</p>
<p>As i mentioned, it’s not supported to trigger a workflow based on a watchlist of repositories releases or changes.</p>
<p>You can raise a feature_request ticket <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">here</a> and discuss with developing team.</p>
<p>Thanks.</p>

**Comment by yonicd** [comment]
<p>I posted it to the link  you supplied.</p>
<p>Thanks.</p>
